docs/blog/newsletter-june-2021.rst

```diff
@@ -0,0 +1,137 @@
+
+.. post:: June 03, 2021
+   :tags: newsletter
+
+#####################################
+Write the Docs Newsletter – June 2021
+#####################################
+
+Hi everybody! It's Beth here, writing to you from my gloriously sunny balcony (thank *goodness* rainy May is over). I hope you're all enjoying the weather wherever you are.
+
+The big thing I want to shout about this month is the `Prague Call for Proposals </conf/prague/2021/cfp/>`__. If you were even sort-of entertaining the thought of submitting a talk idea, take this as your sign to go for it! My favourite thing about giving conference talks is the conversations it starts. It's awesome for helping you find people who care about the same things as you; so if there's something you'd really like to see a talk about this year, I really encourage you to submit one.
```

Comment:
"is the conversation" it starts ... or "are the conversations" it starts ... yes?

Reply:
fixed the mixed singular/plural